menas/src/main/scala/za/co/absa/enceladus/menas/repositories/OozieRepository.scala

```diff
@@ -188,6 +216,60 @@ class OozieRepository @Autowired() (oozieClientRes: Either[OozieConfigurationExc
         classLoader.getResourceAsStream(fileName), "UTF-8")).lines().toArray().mkString("\n")
   }
 
+  /**
+   * This is a helper function for impersonating oozie calls using the proper proxy user if configured
+   * 
+   * @param user User to impersonate
+   * @fn Oozie action to perform - Important to note that this should be Oozie action only (only wrap the call to oozieclient)
+   */
+  private def impersonateWrapper[T](user: String)(fn: () => T) = {
+    if (oozieProxyUser.isEmpty || oozieProxyUserKeytab.isEmpty) {
```

Comment:
BTW this is being called every time someone touches schedulers action buttons. Which means it could write too much to the catalina log.

Reply:
Maybe we can change the log level here?

The kerberos log can be turned off by turning off krb5dbg